project-base/composer.json

```diff
@@ -109,11 +109,11 @@
     "require-dev": {
         "ext-pgsql": "*",
         "ext-zip": "*",
-        "codeception/codeception": "2.4.0",
+        "codeception/codeception": "^2.4.0",
```

Comment:
There were 3 changes in the monorepo's composer.json but only 1 change outside of it. Seems that you missed something...